txtdirect_test.go

```diff
@@ -454,3 +454,39 @@ func TestRedirectE2e(t *testing.T) {
 		}
 	}
 }
+
+func TestConfigE2e(t *testing.T) {
+	tests := []struct {
+		url    string
+		txt    string
+		enable []string
+	}{
+		{
+			"https://e2e.txtdirect",
+			txts["_redirect.path.txtdirect."],
+			[]string{},
+		},
+		{
+			"https://path.txtdirect/test",
+			txts["_redirect.path.e2e.txtdirect."],
+			[]string{"host"},
+		},
+		{
+			"https://gometa.txtdirect",
+			txts["_redirect.gometa.txtdirect."],
+			[]string{"host"},
```

Comment:
Shouldn't that be `gometa` being enabled instead of `host`?

Reply:
Since they are negative tests, we should enable wrong options.

Reply:
Ah true. Sorry totally missed that. Happy to merge and waiting for positive tests in a follow on PR.